angelz510/react-challenge-project-sep-trial
# fix(logout): fixed logout button to redirect and clear login info on redux

## Changes
1. implemented useDispatch to dispatch logout function in the label 'log out' to give the button functionality.
2. imported useDispatch and logout form authActions.
3. added redux devtools to store to make sure auth was cleared after logout. 

## Purpose
Problem was that the logout button out redirect user to login but would not clear auth state. 

## Approach
This change logs user out on redux on click.

## Testing Steps
log in, open redux devtools, look at auth, log out, state should go back to null.

## Learning
was unable to check if users info would clear after log out so I had to implement redux devtools in store.

Feature Request: Log Out
Closes #005
